src/Orchard/DisplayManagement/Descriptors/ShapePlacementStrategy/ShapePlacementParsingStrategy.cs

```diff
@@ -111,8 +111,9 @@ public class ShapePlacementParsingStrategy : IShapeTableProvider {
             differentiator = "";
             shapeType = shapeLocation.ShapeType;
             var dashIndex = shapeType.LastIndexOf('-');
-            if (dashIndex > 0 && dashIndex < shapeType.Length - 1) {
-                differentiator = shapeType.Substring(dashIndex + 1);
+            var dashIndex = shapeType.LastIndexOf("__");

```

Comment:
You are right, I created the pull request copy pasting my code to the github web editor, but I didn't notice I left original dashIndex declaration in web editor. 
For future pull requests I will configure my PlasticSCM client to connect directly to the git repository to create my pull requests in order to avoid this kind of mistakes..